types/test/test_chronology.c

```diff
@@ -32,7 +34,8 @@ int error_context(rage_Error (*f)(StringBuffer b)) {
         return RAGE_ERROR(eb.buf); \
     }
 
-rage_Error test_time_delta(StringBuffer eb) {
+// FIXME: this seems to be the only thing exercising this code, remove?
```

Comment:
As in, the code is not used in production, just by this test?

Reply:
Yes, I wrote all the time helpers quite early and ended up not really using many of them at all.